README.md

```diff
@@ -57,7 +57,7 @@ Use a package manager:
 
 ```bash
 # macOS
```

Comment:
would be good to update this reference to `Homebrew (macOS/Linux)`

Reply:
Done!